### PR TITLE
Updating rubocop

### PR DESCRIPTION
**Pull request by jeremyf**

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -2,6 +2,11 @@
 require: rubocop-rspec
 inherit_from: .rubocop_todo.yml
 
+# With the rubocop 0.47.0 and rubocop-rspec 1.8.0 the following stderr message was sent:
+#   An error occurred while RSpec/DescribedClass cop was inspecting path/to/file
```
Comment (jcoyne) [inline .rubocop.yml]: Is this a bug in rubocop?  Is there a ticket open for it?

File: .rubocop_todo.yml [hunk 2]
```diff
@@ -1,18 +1,32 @@
...
+# Include: **/Gemfile, **/gems.rb
+Bundler/OrderedGems:
   Exclude:
-    - 'spec/models/solr_endpoint_spec.rb'
+    - 'Gemfile'
+
+# Offense count: 56
+# Configuration parameters: CountComments, ExcludedMethods.
+Metrics/BlockLength:
+  Max: 220
```
Comment (jcoyne) [inline .rubocop_todo.yml]: This seems really big, possibly it's an outlier (config block maybe?) Do we want to exclude just a few files and reduce the max?

File: Gemfile [hunk 3]
```diff
@@ -46,8 +46,8 @@ group :development, :test do
   gem 'solr_wrapper', '~> 0.10'
   gem 'fcrepo_wrapper', '~> 0.4'
 
-  gem 'rubocop', '~> 0.43.0'
-  gem 'rubocop-rspec', '~> 1.8.0'
+  gem 'rubocop', '~> 0.43'
```
Comment (jcoyne) [inline Gemfile]: This seems like it'll break next time rubocop is updated.  I'd prefer we handle the updates on our terms.

File: .rubocop.yml (hunk 1 above)
Comment (jeremyf) [inline .rubocop.yml]: I think the issue is that 0.47.0 was released and 1.10.0 of rubocop-rspec was released. So we were running into conflicts in API changes (or some such thing).

**Pull request merged**